### DISCUSSION TITLE
[10 Giugno 2022] - Cache

### DISCUSSION THREAD

**Discussion by alem1105** [CS-Swap/Architettura-degli-Elaboratori#14] [Cache]
### Traccia Esercizio

![immagine](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/1fdd8d88-fac7-4d8f-9d58-d6c3eeaba355)

<details><summary>Soluzione Prof</summary>
<p>

![immagine](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/48867964/3c0582d4-15c9-4d50-8bad-9d8731a20a90)

</p>
</details> 

>**[[Altre Soluzioni](https://github.com/sapienzastudentsnetwork/architettura-degli-elaboratori/discussions/55)]**

**Comment by notedo** [comment]
![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/f9ce525e-ad74-4f80-99f3-d3d5724bb590)
